impl/src/bin/cargo-raze.rs

```diff
@@ -134,6 +134,8 @@ fn main() -> Result<()> {
 
       bazel_renderer.render_remote_planned_build(&render_details, &planned_build)?
     }, /* exhaustive, we control the definition */
+    // Settings should always have `genmode` set to one of the above fields
+    GenMode::Unspecified => unreachable!(),
```

Comment:
We can probably just return empty list here. In fact, that might actually be an interesting behavior because it means we've just done planning but we're not going to write anything.

(i'm going to make roughly the same comment everywhere)